<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CI_Edit</class>
 <widget class="QDialog" name="CI_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="contenedor">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tomar Fotograma</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="table"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="varToChange">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>102</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="varToRename">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tomar">
             <property name="text">
              <string>Tomar CI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Editar Fotograma</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="obtener">
             <property name="text">
              <string>Obtener CI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="table2"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="varToChange_2">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="eliminar">
             <property name="text">
              <string>Eliminar CI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="varToRename_2">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="renombrar">
             <property name="text">
              <string>Renombrar CI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
